src/components/context_menu.tsx

```diff
@@ -13,6 +13,8 @@ export interface ContextMenuProps {
   menuOffsetDirection?: MenuOffsetDirection;
   /** Direction to position the menu: 'north' | 'south' */
   menuDirection?: MenuDirection;
+  /** Whether the context menu is position in a panel, such as a header or footer */
+  menuPanelPosition?: boolean;
```

Comment:
Can't we set the correct CSS rules automatically by checking if it's a child element inside a panel component?

Reply:
This doesn't sound like a `boolean` property.